docs/modules/deployment/pages/upgrade/debian/up-debian.adoc

```diff
@@ -52,17 +51,17 @@ sudo apt-mark hold libopennms-java \
 +
 [source, console]
 ----
-apt-get install java-11-openjdk java-11-openjdk-devel
+apt-get install openjdk-11-jdk
```

Comment:
There is a note about using the "headless" command as we do in Minion, but that seems to be related to CentOS/RHEL 8. Ubence might be able to clarify.

Reply:
I can, the package names java-11-openjdk and java-11-openjdk-devel are RHEL/CentOS specific package names. The openjdk-11-jdk package is the Debian/Ubuntu speific package name. Running the command as it is with the RHEL/CentOS names will generate an error.

Reply:
You now have me second guessing myself. I just checked my Ubuntu system that I have this installed on and here's what I'm seeing when running apt list openjdk*:
`openjdk-11-jdk-headless/jammy-updates,jammy-security,now 11.0.16+8-0ubuntu1~22.04 amd64 [installed]
openjdk-11-jdk/jammy-updates,jammy-security 11.0.16+8-0ubuntu1~22.04 amd64
openjdk-11-jre-dcevm/jammy 11.0.12+7-1 amd64
openjdk-11-jre-headless/jammy-updates,jammy-security,now 11.0.16+8-0ubuntu1~22.04 amd64 [installed,automatic]
openjdk-11-jre-zero/jammy-updates,jammy-security 11.0.16+8-0ubuntu1~22.04 amd64
openjdk-11-jre/jammy-updates,jammy-security,now 11.0.16+8-0ubuntu1~22.04 amd64 [installed,automatic]`
I don't have any logs from the initial install, so I'm now not entirely sure what packages got pulled.
Let me spin up a Ubuntu instance real quick and double check. It might not even be the jdk packages based on the above.
I'll respond back when I know more.

Reply:
Sorry for the wait, I got a Ubuntu 22.04 server up and running and got OpenNMS installed, and it wants these two packages specifically for the install:
```
openjdk-11-jdk-headless
openjdk-11-jre-headless
```
So it would be those two for certain that should be in the upgrade command just to be consistent with what got installed.

Reply:
```suggestion
apt-get install openjdk-11-jdk-headless openjdk-11-jre-headless
```